docker-compose.yml

```diff
@@ -217,6 +217,33 @@ services:
     volumes:
       - devpi_data:/data
 
+  edraak_programs:
+    image: edraak-devstack-programs
+    environment:
+      DJANGO_SETTINGS_MODULE: edraakprograms.dev
+      PROGS_CFG: '/app/docker.json'
+      NODE_ENV: development
+      MYSQL_ROOT_PASSWORD: ''
+      MYSQL_DATABASE: 'edraakprograms'
+    command: bash -c '/app/docker-entry-point.sh edx.devstack.mysql 3306 python manage.py runserver 0.0.0.0:8000'
+    container_name: edraak.devstack.programs
+    working_dir: /app
+    ports:
+      - "18901:8800"
+    depends_on:
+      - mysql
+      - edraak_programs_mongodb
+      - memcached
+
+  edraak_programs_mongodb:
```

Comment:
Use the same mongo database for edxapp, no need for a new database.

Reply:
Done